src/update_agent/actor.rs

```diff
@@ -135,7 +145,7 @@ impl UpdateAgent {
     }
 
     /// Initialize the update agent.
-    fn initialize(&mut self) -> ResponseActFuture<Self, Result<(), ()>> {
+    fn tick_initialize(&mut self) -> ResponseActFuture<Self, Result<(), ()>> {
```

Comment:
Why the naming changes? Is it to represent that these functions are directly called by the tick handler?

Reply:
Yes (but mostly: a drive-by change to make code easier to skim/grep).